functional/FunctionalTest.java

```diff
@@ -421,6 +422,34 @@ private static void handleException(String methodName, String args, long startTi
     throw e;
   }
 
+  // private static void deleteFilesAndObjects(String bucketName, String[] filenames) throws
+  // Exception {
+  //   for (String filename : filenames) {
+  //     Files.delete(Paths.get(filename));
+  //
+  // client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename).build());
+  //   }
+  // }
+
+  private static void deleteFilesAndObjects(String bucketName, String[] filenames)
+      throws Exception {
+    for (String filename : filenames) {
+      Files.delete(Paths.get(filename));
+    }
+    List<DeleteObject> objects =
```

Comment:
1. Can't you use `removeObjects()` method available in this class?
2. Its better to call object removal first, then file removal later.